e2e_deployment_files/long_haul_deployment_constrained.template.json

```diff
@@ -0,0 +1,436 @@
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "env": {
+                            "transportType": {
+                                "value": "<ClientModuleTransportType>"
+                            },
+                            "inputName": {
+                                "value": "input1"
+                            },
+                            "outputName": {
+                                "value": "output1"
+                            },
+                            "testResultCoordinatorUrl": {
+                                "value": "http://testResultCoordinator:5001"
+                            }
+                        },
+                        "settings": {
+                            "image": "<Container_Registry>/microsoft/azureiotedge-relayer:<Build.BuildNumber>-linux-<Architecture>",
+                            "createOptions": "{\"HostConfig\":{\"Memory\":\"<RelayerMemoryThreshold>\", \"LogConfig\":{\"Type\":\"json-file\",\"Config\":{\"max-size\":\"4m\",\"max-file\":\"<LogRotationMaxFile>\",\"compress\":\"true\"}}}}"
+                        }
+                    },
+                    "twinTester1": {
+                        "version": "1.0",
+                        "type": "docker",
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "env": {
+                            "TransportType": {
+                                "value": "<ClientModuleTransportType>"
+                            },
+                            "trackingId": {
+                                "value": "<TrackingId>"
+                            },
+                            "testResultCoordinatorUrl": {
+                                "value": "http://testResultCoordinator:5001"
+                            },
+                            "testStartDelay": {
+                                "value": "<TestStartDelay>"
+                            },
+                            "ServiceClientConnectionString": {
+                                "value": "<IoTHubConnectionString>"
+                            },
+                            "TwinUpdateSize": {
+                                "value": "<TwinUpdateSize>"
+                            },
+                            "TwinUpdateFrequency": {
+                                "value": "<TwinUpdateFrequency>"
+                            },
+                            "EdgeHubRestartFailureTolerance": {
+                                "value": "<EdgeHubRestartFailureTolerance>"
+                            },
+                            "TwinUpdateFailureThreshold": {
+                                "value": "<TwinUpdateFailureThreshold>"
+                            },
+                            "StorageOptimizeForPerformance": {
+                                "value": "<OptimizeForPerformance>"
```

Comment:
It doesn't feel like this should be the ONLY spot with `StorageOptimizeForPerformance` for this file.

Reply:
The twin tester, edge hub, and the TRC are the only ones that use rocksdb right? Edge Agent does too, but not sure if it exposes this config.

Where else did you want me to add `OptimizeForPerformance`?